src/txmempool.cpp

```diff
@@ -965,7 +966,14 @@ void CTxMemPool::queryHashes(std::vector<uint256>& vtxid) const
 }
 
 static TxMempoolInfo GetInfo(CTxMemPool::indexed_transaction_set::const_iterator it) {
-    return TxMempoolInfo{it->GetSharedTx(), it->GetTime(), it->GetFee(), it->GetTxSize(), it->GetModifiedFee() - it->GetFee()};
+    // ELEMENTS: include the discounted vsize of the tx
+    size_t discountvsize = it->GetTxSize();
+    CTransaction tx = it->GetTx();
```

Comment:
In 8bafd0c9fa448c3790ed2992f02b8e1245e6c56a:

This variable assignment should be moved inside the `if` block.

Reply:
unnecessary after 6c67c5e842ef4c9db22095b5bfcdccb2add12540